B/basiclu/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "basiclu"
+version = v"2.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ERGO-Code/basiclu.git", "5c882bd2617006d0d1ee6ba3400686da092ecd85"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+if [[ ${target} == *mingw32* ]]; then
+    for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+        atomic_patch -p1 ${f}
+    done
+    UNAME="Windows"
+else
+    UNAME="$(uname)"
+fi
+cd basiclu/
+if [[ "${target}" == *-apple-* ]]; then
+    CFLAGS="-D_DARWIN_C_SOURCE"
+elif [[ "${target}" == *-linux-* ]]; then
+    LDLIBS="-lm -lrt"
+fi
+if [[ "${target}" == *freebsd* ]]; then
+    make CC99="cc -std=c99" CFLAGS="${CFLAGS}" LDLIBS="${LDLIBS}" UNAME="${UNAME}" -DCLOCK_MONOTONIC_RAW=CLOCK_MONOTONIC
```

Comment:
`-D` is a compiler flag to set macros on the command line. Make doesn't know anything about that, you can't pass it that way. This Makefile accepts compiler flags with the variable `CFLAGS` so you can do the same as macOS above